app/jobs/hackathons/archive_website_job.rb

```diff
@@ -0,0 +1,11 @@
+class Hackathons::ArchiveWebsiteJob < ApplicationJob
+  def perform(hackathon)
+    # This request can take a long time!
+
+    # Web Archive will only save status 200 pages by default
+    website = hackathon.website.sub(/^https?:\/\/(www.)?/, "")
```

Comment:
@sampoder just wondering, is there a reason why we're removing the scheme and `www` from the URL?

Reply:
I think for the web archive API you need to remove https